client/src/components/Settings/SettingsView.res

```diff
@@ -39,7 +39,10 @@ let tabTitleView = (tab: T.Tab.t): Html.html<msg> => {
 
     Html.h3(
       list{
-        Attrs.classList(list{("tab-title", true), ("selected", isSameTab)}),
+        Attrs.classList(list{
+          (%twc("h-full m-0 mb-[-3px] px-2.5 py-0 text-grey2 cursor-pointer"), true),
```

Comment:
I'd rather not use the `[]` syntax, especially with pixels, as tailwind AFAIK uses `rem`s which support zoom better.

To remove them, we can either add new presets, or if it looks the same either way (eg often 1 pixel doesn't matter), just round them to the nearest existing preset.

Reply:
(this comment applies to anywhere that `[]` is used)